examples/fuse_examples/imaging/classification/cmmd/runner.py

```diff
@@ -319,15 +325,15 @@ def main(cfg : DictConfig) -> None:
     # 2. folder named CMMD which is the downloaded data folder
 
     # train
-    if 'train' in RUNNING_MODES:
-        run_train(cfg["paths"] ,cfg["train"])
+    if 'train' in cfg["run.running_modes"]:
+        model, trainer = run_train(cfg["paths"] ,cfg["train"])
 
     # infer
-    if 'infer' in RUNNING_MODES:
-        run_infer(cfg["paths"] , cfg["infer"])
+    if 'infer' in cfg["run.running_modes"]:
+        run_infer(model, trainer, cfg["paths"] , cfg["infer"])
```

Comment:
if "train" not in running modes - you'll get an error.
Please support such a case.

Reply:
added assert

Reply:
But we want to support it.
Why don't you create the model and trainer in run_infer?